microsoft-edge/devtools-guide-chromium/memory-inspector/memory-inspector-tool.md

```diff
@@ -21,48 +22,176 @@ ms.date: 02/15/2022
+ 
+    * From the context menu. Right click on the `buffer` property and select **Reveal in Memory Inspector panel**.
 
+    ![Open Memory Inspector from the buffer property line context menu](../media/memory-inspector-open-from-buffer.png)
 
-Use the new **Memory Inspector** tool to inspect an `ArrayBuffer` in JavaScript and Wasm memory.
+    The JavaScript ArrayBuffer opens in Memory Inspector.
 
-1. Open the demo webpage [Inspect ArrayBuffers in JS (Memory in JS)](https://memory-inspector.glitch.me/demo-js.html) in a new window or tab.
+    ![ArrayBuffer open in the Memory Inspector panel](../media/memory-inspector-panel.png)
+
+### Inspect multiple objects
+
+You can inspect multiple objects at the same time such as DataView and TypedArray.
+
+In the demo webpage line `b2` is a TypedArray. Right click on the `b2` property and select **Reveal in Memory Inspector panel**.
+
+A new tab opens next to the first tab in Memory Inspector.
+
+![Two ArrayBuffer tabs open in the Memory Inspector panel](../media/memory-inspector-panel-two.png)
+
+<!-- ====================================================================== -->
+## Navigating in Memory Inspector
 
-   The following list of steps is derived from the instructions on that webpage.
+The Memory Inspector panel includes three types of content:
 
-1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
+* [Navigation bar](#navigation-bar)
+* [Memory buffer](#memory-buffer)
+* [Value inspector](#value-inspector)
 
-1. In DevTools, on the main toolbar, select the **Sources** tab.  If that tab isn't visible, click the **More tabs** (![More tabs icon.](../media/more-tabs-icon-light-theme.png)) button, or else the **More Tools** (![More Tools icon.](../media/more-tools-icon-light-theme.png)) button.
+### Navigation bar
 
-1. In the **Page** tab on the left, select the file `demo-js.js`. <!-- `memory-write-wasm`-->
+![Memory Inspector panel Navigation bar](../media/memory-inspector-panel-navigation-bar.png)
+ 
+* The address input shows the current byte address in hex format. You can change the value to jump to a new location in the memory buffer. Click in the text box and change the value to `0x00000008`. The Memory buffer immediately jumps to that byte address.
 
-1. Set a breakpoint at line 18 in the loop body.
+* Memory buffers may be longer than one page. Use the left and right arrow buttons to navigate Previous page and Next page, respectively. If there is only one page of memory buffer data, the arrows take you to the beginning and ending of the page.
 
-1. Refresh the webpage.
+* Use the far-left history arrows to **Go back in address history** and **Go forward in address history**.
```

Comment:
It took me awhile to understand what arrows you were talking about here. I would include an icon with the arrows so it's easier to identify them or annotate the screenshot with numbers and then use the numbers to reflect the part of the UI you're talking about here

Reply:
done